core/trino-main/src/test/resources/cert/generate.sh

```diff
@@ -2,6 +2,16 @@
 
 set -eux
 
-openssl req -new -x509 -newkey rsa:4096 -sha256 -nodes -keyout localhost.key -days 3560 -out localhost.crt -config localhost.conf
-cat localhost.crt localhost.key > localhost.pem
+script_dir=$(CDPATH="" cd -- "$(dirname -- "$0")" && pwd)
+outfile=$1
+shift
+if [ -z "$outfile" ]; then
+    outfile="$script_dir/localhost.pem"
```

Comment:
how can `outfile` be empty? is this called with `""` param? that's a weird convention. Maybe we can make $1 simply required?

i guess `$1` above would fail when no param at all, right?

Reply:
`outfile` is empty if `$1` is, quotes don't change this. I didn't make this requires to avoid changing the behavior of this script, someone might be used to it and the use case when you specify this param would be very rare.